api/src/libs/migrationUtils.ts

```diff
@@ -0,0 +1,25 @@
+import { BusinessesDataClient } from "@domain/types";
+import { LogWriterType } from "@libs/logWriter";
+import { UserData } from "@shared/userData";
+
+export async function pushBusinessesInUsersDataToBusinessTable(
```

Comment:
[dust] Dustiest of dust comments, but should this be called something like `copyBusinessDataFromUsersTableToBusinessesTable`?